test_launch_ros/test/test_launch_ros/frontend/test_node_frontend.py

```diff
@@ -47,13 +48,13 @@ def test_launch_remapping_xml():
                     <param name="param5" value="$(var a_string)"/>
                     <param name="param6" value="2., 5., 8." value-sep=", "/>
                     <param name="param7" value="'2', '5', '8'" value-sep=", "/>
-                    <!-- TODO(hidmic): revisit after https://github.com/ros2/launch_ros/pull/137 gets merged -->
                     <param name="param8" value="&quot;'2'&quot;, &quot;'5'&quot;, &quot;'8'&quot;" value-sep=", "/>
                     <param name="param9" value="\'2\', \'5\', \'8\'" value-sep=", "/>
-                    <!-- TODO(hidmic): revisit after https://github.com/ros2/launch_ros/pull/137 gets merged -->
                     <param name="param10" value="&quot;'asd'&quot;, &quot;'bsd'&quot;, &quot;'csd'&quot;" value-sep=", "/>
                     <param name="param11" value="'\asd', '\bsd', '\csd'" value-sep=", "/>
-                    <param name="param12" value=""/>
+                    <param name="param12" value="''"/>
+                    <param name="param13" value="$(var my_value)" type="str"/>
+                    <param name="param14" value="'2', '5', '8'" value-sep=", " type="list_of_str"/>
```

Comment:
@ivanpauno should it be 
```suggestion
                    <param name="param14" value="2, 5, 8" value-sep=", " type="list_of_str"/>
```
instead, to actually make sure YAML rules won't get in the way? Same below.

Reply:
the expected output is `["'2'", "'5'", "'8'"]`, this is checking that YAML rules aren't used in the way.

Reply:
Checking that YAML rules are not used to parse a list, but not checking that YAML rules are not used for each element.

Reply:
> Checking that YAML rules are not used to parse a list, but not checking that YAML rules are not used for each element.

The first element is `"'2'"`. If yaml rules were being used, the output would be `'2'`. This is checking that the output is `"'2'"`.
If I do the change you ask the test doesn't pass.

Reply:
> If I make the change you ask for, the test won't pass.

How so?